mcf-browser/src/back-end/utils.js

```diff
@@ -103,10 +111,13 @@ function getValueFromValueObj(valueObj) {
  *     Data Commons Knowledge Graph.
  */
 async function doesExistsInKG(dcid) {
-  const url = API_ROOT + '/node/triples?dcids=' + dcid + '&limit=1';
+  const url = API_ROOT + '/v1/property/out/typeOf/values/' + dcid;
```

Comment:
Same comment here: update to /v1/property/values/out/ + dcid + /typeOf

Reply:
Added in next commit!